sale_framework/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Framework",
+    "summary": """Manage blanket order and call of ordr""",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "depends": [
+        "sale_manual_delivery",
+        "sale_stock_prebook",
```

Comment:
These dependencies seem too much for a "framework".

Reply:
It's probably not the best name. I have no other idea as the name “sale_blanket_order” is already in use.

Reply:
Yeah, but it's not only a question of the name, but to require such dependencies, that I think they should be extension modules: `sale_framework_manual_delivery` and `sale_framework_stock_prebook`. It also highlights that even using `sale.order` as base, it requires you some glue code for making it work. I would include also `order` on the name to be more specific. It's true that is not easy to name this. What about `sale_order_multi_purpose`?